webapp/src/index.js

```diff
@@ -23,4 +22,4 @@ class Plugin {
     }
 }
 
-window.registerPlugin(pluginId, new Plugin());
+window.registerPlugin(pluginId, new Plugin());
```

Comment:
Can you please revert changes here so this file is not changed in this PR?

Reply:
There are still two changes in this file that should be removed to simplify the diff in this file.

- A newline should be added back to the end of this file
- Line 7 should be added back